content/applications/marketing/email_marketing/lost_leads_email.rst

```diff
@@ -0,0 +1,321 @@
+- A pair of **Active** filters to target *both* active and inactive leads
+
+Create a lost leads reactivation email
+======================================
+
+1. First, navigate to the *Email Marketing* app, and on the :guilabel:`Mailings` page, click the
+   :guilabel:`New` button in the top-left corner.
+
+   On the new :guilabel:`Mailings` form, enter an appropriate :guilabel:`Subject` line for the email
+   in the corresponding field. Then in the :guilabel:`Recipients` field, choose the
+   :guilabel:`Lead/Opportunity` model from the drop-down menu.
+
+2. In the rules section, click the modify filter (:guilabel:`▶ (triangle pointing right)`) icon,
+   below the :guilabel:`Recipients` field, to expand the filter rules. Leave the default
+   :guilabel:`Blacklist` rule in place, and then click :guilabel:`New Rule`.
+
+3. Add a :guilabel:`Created on` parameter in the new rule's first field, from the drop-down menu, to
+   designate a specific time period during which the targeted leads were lost (i.e. 30 days prior,
+   90 days prior, previous year, etc.).
+
+   Then, in the second field, select a date operator :guilabel:`<= (less than or equal to)`,
+   :guilabel:`>= (greater than or equal to)`, or :guilabel:`is between` in order to frame the time
+   selection chosen in the third field.
+
+   In the third field, use the calendar popover window to select dates, and click :guilabel:`Apply`
+   to lock in the time range.
+
+   .. image:: lost_leads_email/created-on.png
+      :align: center
+      :alt: A custom filter rule setting the time period to be anything before today's date.
+
+.. important::
+   Now that there is more than one rule applied, make sure the statement at the top of
+   :guilabel:`Recipients` filter list reads :guilabel:`Match all of the following rules`. If it does
+   not, click on the statement, and select :guilabel:`all` from the drop-down menu (as opposed to
+   :guilabel:`any`).
+
+   .. image:: lost_leads_email/match-all.png
+      :align: center
+      :alt: The statement at the top of the filters list, with the drop-down menu open.
+
+4. Now add :guilabel:`Stage` filters to exclude leads in the *New*, *Qualified*, and *Won* stages of
+   the sales pipeline.
+
+   .. note::
+      This step assumes that the *New*, *Qualified*, and *Won* stages exist in the CRM pipeline,
+      however stage names may differ from business to business. Refer to the database's actual
+      stage names in the *CRM* app's pipeline to complete this step, accordingly.
+
+   Begin again by clicking :guilabel:`New Rule` and selecting :guilabel:`Stage` from the first
+   field's drop-down menu. In the second field, select the :guilabel:`does not contain` operator,
+   and in the third field, type `new` as the rule's defining parameter.
+
+   Next, click the :guilabel:`➕ (plus)` icon to the right of the rule's row to duplicate it, and in
+   the new rule's third field, delete :guilabel:`new`, and enter :guilabel:`qualified`. After that
```

Comment:
Rewrote the section here to just use the **is in** vs. **contains** operator since it's easier + more efficient to do. Should help clean up the gui problem here and in other sections throughout the doc.